libmscore/score.cpp

```diff
@@ -2295,11 +2295,19 @@ bool Score::appendScore(Score* score)
             return false;
       int tickOfAppend = lastMeasure->endTick();
 
-      if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak()) {
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      if (addPageBreak) {
+            // add PageBreak if one doesn't already exist
+            if (!lastMeasure->pageBreak())
```

Comment:
I'm looking over this line and am thinking maybe I should do <pre>if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak())</pre>instead, so that only a Line or Page break will be the result, not both Line & Page break.